docker/yb-voyager-docker

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+argv=( "$@" )
+
+i=0
+while [ $i -lt $# ]
+do
+    if [[ ${argv[${i}]} == "--export-dir" ]]
+    then
+        j=$(( $i + 1))
+        vol="-v ${argv[$j]}:/export-dir"
+        argv[$j]='/export-dir'
+    fi
+    i=$(( $i + 1))
+done
+
+dockerCmd="docker run ${vol} -it --network=host --rm --privileged localhost/yb-voyager:0.82 yb-voyager ${argv[*]}"
```

Comment:
Use `latest` tag instead of `0.82`. We don't want to update this script after every release.
Why do we need the `--network=host`?

Reply:
Used the latest tag